File: PRAK204-2310817120012-RirinCitraLestari.c
#include <stdio.h>
#include <math.h>

int main() {
    double jari_jari, tinggi, volume, luas, keliling;

    scanf("%lf", &jari_jari);
    scanf("%lf", &tinggi);

    volume = M_PI * jari_jari * jari_jari * tinggi;
    luas = 2 * M_PI * jari_jari * (jari_jari + tinggi);
    keliling = 2 * M_PI * jari_jari;

    printf("Volume = %.2lf\n", volume);
    printf("Luas = %.2lf\n", luas);
    printf("Keliling = %.2lf\n", keliling);

    return 0;
}

File: PRAK204-2310817120012-RirinCitraLestari.py
import math

jari_jari, tinggi = map(float, input().split())
volume = math.pi *jari_jari**2 * tinggi
luas = 2 * math.pi * jari_jari * (jari_jari + tinggi)
keliling = 2 * math.pi * jari_jari 
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")